data/incipias/incipias first contact.txt

```diff
@@ -193,6 +193,8 @@ mission "Incipias: The Derelict"
 
 mission "Incipias: Help The Stranded 1"
 	landing
+	name "Go to the Incipias"
+	description "Go to the Incipias planet <planet>, in the <system> system, to see what they have to say about helping a disabled Incipias ship."
```

Comment:
```suggestion
	description "Ask the Incipias for help to save one of their ships."
```
The Quarg don't tell you where to go to, and they are quite reluctant to let you go at all. Also, you land on a station, not a planet. And even if you landed on a planet, `<destination>` would be the appropriate way to say it.

Reply:
(but I wanted a comma)